### PR TITLE
fix(client): drawer positions

### PR DESCRIPTION
**Pull request by alexzhang1030**
fixes this:

https://github.com/vuejs/devtools-next/assets/49969959/539789cf-0cef-4eea-be93-60aec0d025c1

### CODE REVIEW

File: packages/ui/src/components/Overlay.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@ defineProps<OverlayProps>()
 
 <template>
   <div
-    class="fixed left-0 top-0 $ui-z-max h-full w-full $ui-fcc transition-opacity transition-duration-300 $ui-bg-base"
+    class="absolute left-0 top-0 $ui-z-max h-full w-full $ui-fcc transition-opacity transition-duration-300 $ui-bg-base"
```
Comment (webfansplz) [inline packages/ui/src/components/Overlay.vue:12]: Why did we change the position type, I think keeping it as fixed position type will be more flexible and easier to use.

Comment (alexzhang1030) [reply packages/ui/src/components/Overlay.vue:12]: fixed has an issue，see the picture below

image

the top or left will be calc by viewport，that will cause position error on split screen.

Maybe we can make absolute as an option？

Comment (webfansplz) [reply packages/ui/src/components/Overlay.vue:12]: Sure, we can add an option to handle it.

Comment (alexzhang1030) [reply packages/ui/src/components/Overlay.vue]: Added

File: packages/ui/src/components/Overlay.vue [hunk 2]
```diff
@@ -2,17 +2,26 @@
...
+   */
+  position?: 'fixed' | 'absolute'
 }
 
-defineProps<OverlayProps>()
+const props = withDefaults(defineProps<OverlayProps>(), {
+  position: 'fixed',
+})
+
+console.log({ props })
```
Comment (webfansplz) [inline packages/ui/src/components/Overlay.vue:15]: Let's remove the console output

Comment (alexzhang1030) [reply packages/ui/src/components/Overlay.vue]: Sry 😂

**Review by webfansplz — APPROVED** [review]

**Pull request merged**